api/v1/db/json/pets-dao-example.js

```diff
@@ -46,4 +46,57 @@ const getPetById = id => new Promise((resolve, reject) => {
   }
 });
 
-module.exports = { getPets, getPetById };
+/**
+ * Posts a new pet
+ *
+ * @param {object} body Request body
+ * @returns {Promise} Promise object represents ?
+ */
+const postPet = body => new Promise((resolve, reject) => {
```

Comment:
Could you use async/await notation here instead of the `Promise` constructor? This should simplify the code. Could you also update the other dao methods to use this syntax?